### PR TITLE
Prep for 2.1.0 release

### PR DESCRIPTION
**Pull request by ferventcoder**

### CODE REVIEW

**Comment by cyberious** [comment]
@ferventcoder needs rebase

**Comment by ferventcoder** [comment]
Yes it does.

**Comment by ferventcoder** [comment]
I'm not completely sure what happened here

**Comment by ferventcoder** [comment]
I pushed to what I thought was my stable, but apparently it was hooked up to upstream (this repo). :/ We'll need to merge this and then add more to it as a separate PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+##2015-08-31 - Release 2.1.0
+### Summary
+
+Add Chocolatey to the module pack
```
Comment (ferventcoder) [inline CHANGELOG.md]: This would need updated with everything that would go in for the release.

**Pull request closed without merging**